app/src/main/java/fr/free/nrw/commons/upload/categories/UploadCategoriesFragment.java

```diff
@@ -198,6 +198,19 @@ public void setCategories(List<CategoryItem> categories) {
         } else {
             adapter.setItems(categories);
         }
+        adapter.notifyDataSetChanged();
```

Comment:
Could you please add a `//` comment here explaining briefly what the following block does?
It will save time to the many people who will read the code in the future.
Thank you! :-)